lib/TPP/ToBlockLayoutAndBack.cpp

```diff
@@ -324,9 +333,51 @@ mlir::linalgx::packConv2DNchwFchwOp(RewriterBase &rewriter,
   return packConvolutions(rewriter, convOp, tiles);
 }
 
+template <typename OpTy>
+static std::pair<SmallVector<AffineMap>, SmallVector<utils::IteratorType>>
+getAffineMapsAndIteratorsForBlockedMatmul(OpTy matmulOp) {
```

Comment:
This isn't just for blocked matmuls.

Reply:
`getAffineMapsAndIteratorsForBlockedMatmul` -> `getAffineMapsAndIteratorsForBlockedMatmulOrBatchMatmul`?